src/components/dialogs/contingency-list/explicit-naming/explicit-naming-form.jsx

```diff
@@ -182,6 +182,7 @@ const ExplicitNamingForm = () => {
                 fileHeaders: csvFileHeaders,
                 getDataFromCsv: getDataFromCsvFile,
                 csvData: csvInitialData,
+                config: { delimiter: ',' },
```

Comment:
Why fixing the delimiter to ',' ?
We should used `delimitersToGuess` prop and juste remove the one we use for multiple values here '|'
see https://react-papaparse.js.org/docs#config : 
```diff
-delimitersToGuess: [',', '	', '|', ';', RECORD_SEP, UNIT_SEP]
+delimitersToGuess: [',', '	', ';', RECORD_SEP, UNIT_SEP]
```

Reply:
And we could fix this rule directly in commons-ui to avoid knowing papaparse here.